.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: nf-core CI
+# This workflow is triggered on pushes and PRs to the repository.
+# It runs the pipeline with the minimal test dataset to check that it completes without any syntax errors
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        # Nextflow versions: check pipeline minimum and current latest
+        nxf_ver: ['19.10.0', '']
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install Nextflow
+        run: |
+          export NXF_VER=${{ matrix.nxf_ver }}
+          wget -qO- get.nextflow.io | bash
+          sudo mv nextflow /usr/local/bin/
```

Comment:
The docker pull is missing from the template I know but that should be done here too (also the tagging...). I believe we didn't add this yet to the template which we might have to do soon.

Reply:
Yep. Will probably need to be added properly to the linting and bump versions too right? Seems to be working for now though 💯 I guess I can wait until we release tools to add this in via the syncing. Going to be off for a month or so now anyway...

Reply:
https://github.com/nf-core/tools/pull/472 will cover this :+1: 

Yeah, I guess most people will be off work now anyways ...

Reply:
Ah, ok. Thats what we were waiting on to release `tools` 👍

Reply:
Ok. Fixed both points relative to that PR. Will update if things change later too. Thanks @apeltzer 👍

Reply:
Fixed in `chipseq` and the other open PRs too. They should all be pretty much the same.